universe/ValueRef.cpp

```diff
@@ -1841,6 +1841,18 @@ double ComplexVariable<double>::Eval(const ScriptingContext& context) const
         return part_type->Capacity();
 
     }
+    else if (variable_name == "PartSecondaryStat") {
```

Comment:
this bit's off topic, should be picked into master at some point if this isn't merged soon